packages/vitest/src/node/config.ts

```diff
@@ -699,6 +699,16 @@ export function resolveConfig(
     port: defaultBrowserPort,
   }
 
+  // enable includeTaskLocation by default in UI mode
+  if (resolved.browser.enabled) {
+    if (resolved.browser.ui) {
+      resolved.includeTaskLocation ??= true
+    }
+  }
+  else if (resolved.ui) {
+    resolved.includeTaskLocation ??= true
+  }
```

Comment:
Does this work for `--reporter=html`?

Reply:
Won't work since source tab is missing when running `pnpm vite preview --outDir=html`, maybe we should add the tab in `readonly` mode